src/actions/breakpoints.js

```diff
@@ -82,6 +128,47 @@ export function enableBreakpoint(location: Location) {
   };
 }
 
+export function syncBreakpoint(sourceId, pendingBreakpoint) {
+  return ({ dispatch, getState, client, sourceMaps }: ThunkArgs) => {
+    const { line, sourceUrl, column } = pendingBreakpoint.location;
+    const location = { sourceId, sourceUrl, line, column };
+    const source = getSource(getState(), sourceId);
+
+    let promise = syncClientBreakpoint(
+      source,
+      client,
+      sourceMaps,
+      pendingBreakpoint,
+      location
+    );
+
+    const breakpoint = _createBreakpoint(location, pendingBreakpoint);
+
+    // If the breakpoint is already disabled, we need to communicate
+    // with the server, but we also have to remove the breakpoint immediately.
+    if (pendingBreakpoint.disabled) {
```

Comment:
why do we need to talk to the server if it's disabled, i think that means that the server doesn't have a bp?